terraform/modules/gsp-cluster/main.tf

```diff
@@ -22,8 +22,10 @@ variable "codecommit_url" {
   type = "string"
 }
 
+data "aws_caller_identity" "current" {}
+
 module "cluster" {
-  source = "git::https://github.com/alphagov/gsp-typhoon//aws/container-linux/kubernetes?ref=gsp"
+  source = "git::https://github.com/alphagov/gsp-typhoon//aws/container-linux/kubernetes?ref=iam_auth"
```

Comment:
This change needs cancelling once https://github.com/alphagov/gsp-typhoon/pull/1 is merged (and should be done before this is merged).